File: webapp-java-spring/src/main/java/de/ehmke/entity/Person.java
package de.ehmke.entity;

import javax.persistence.*;
import java.util.Date;
import java.util.Set;

import static javax.persistence.TemporalType.DATE;

@Entity
@Table(name = "person")
public class Person
{

    /**
     *
     */
    public enum Gender
    {
        MALE, FEMALE
    }

    /**
     * The id
     */
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;

    /**
     * The first name
     */
    @Column(nullable = false, name = "first_name")
    private String firstName;

    /**
     * The last name
     */
    @Column(nullable = false, name = "last_name")
    private String lastName;

    /**
     * The birthday
     */
    @Temporal(DATE)
    @Column(name = "birthday")
    private Date birthday;

    /**
     * The gender
     */
    @Enumerated(EnumType.STRING)
    @Column(nullable = false, name = "gender")
    private Gender gender;

    /**
     * The calendars
     */
    @OneToMany(mappedBy = "person", cascade = CascadeType.REMOVE)
    private Set<Calendar> calendars;

    /**
     * @return the id
     */
    public Long getId()
    {
        return this.id;
    }

    /**
     * @param id the id
     */
    public void setId(Long id)
    {
        this.id = id;
    }

    /**
     * Gets the firstName.
     *
     * @return the firstName
     */
    public String getFirstName()
    {
        return this.firstName;
    }

    /**
     * Sets the firstName.
     *
     * @param firstName the new firstName
     */
    public void setFirstName(String firstName)
    {
        this.firstName = firstName;
    }

    /**
     * Gets the lastName.
     *
     * @return the lastName
     */
    public String getLastName()
    {
        return this.lastName;
    }

    /**
     * Sets the lastName.
     *
     * @param lastName the new lastName
     */
    public void setLastName(String lastName)
    {
        this.lastName = lastName;
    }

    /**
     * Gets the birthday.
     *
     * @return the letzter logout am
     */
    public Date getBirthday()
    {
        return birthday;
    }

    /**
     * Sets the letzter logout am.
     *
     * @param birthday the new letzter logout am
     */
    public void setBirthday(Date birthday)
    {
        this.birthday = birthday;
    }

    /**
     * @return the gender
     */
    public Gender getGender()
    {
        return gender;
    }

    /**
     * @param gender the gender
     */
    public void setGender(Gender gender)
    {
        this.gender = gender;
    }

    /**
     * @return the calendars
     */
    public Set<Calendar> getCalendars()
    {
        return calendars;
    }

    /**
     * @param calendars the calendars
     */
    public void setCalendars(Set<Calendar> calendars)
    {
        this.calendars = calendars;
    }
}

File: webapp-php-symfony/src/AppBundle/Entity/Person.php
<?php
namespace AppBundle\Entity;

use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\ORM\Mapping as ORM;


/**
 * @ORM\Entity(repositoryClass="AppBundle\Repository\PersonRepository")
 */
class Person
{

    /**
     * @ORM\Id
     * @ORM\Column(type="bigint")
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255, nullable=false)
     */
    private $firstName;

    /**
     * @ORM\Column(type="string", length=255, nullable=false)
     */
    private $lastName;

    /**
     * @ORM\Column(type="string", length=1, nullable=false)
     */
    private $gender;

    /**
     * @ORM\Column(type="date", nullable=true)
     */
    private $birthday;

    /**
     * @ORM\OneToMany(targetEntity="AppBundle\Entity\Calendar", mappedBy="owner")
     */
    private $calendars;

    /**
     * Constructor
     */
    public function __construct()
    {
        $this->calendars = new ArrayCollection();
    }

    /**
     * Get id
     *
     * @return integer
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set firstName
     *
     * @param string $firstName
     * @return Person
     */
    public function setFirstName($firstName)
    {
        $this->firstName = $firstName;

        return $this;
    }

    /**
     * Get firstName
     *
     * @return string
     */
    public function getFirstName()
    {
        return $this->firstName;
    }

    /**
     * Set lastName
     *
     * @param string $lastName
     * @return Person
     */
    public function setLastName($lastName)
    {
        $this->lastName = $lastName;

        return $this;
    }

    /**
     * Get lastName
     *
     * @return string
     */
    public function getLastName()
    {
        return $this->lastName;
    }

    /**
     * Get gender
     *
     * @return Gender
     */
    public function getGender()
    {
        return Gender::createInstance($this->gender);
    }

    /**
     * @param Gender $gender
     * @return Person
     */
    public function setGender(Gender $gender)
    {
        $this->gender = $gender->getAbbreviation();

        return $this;
    }

    /**
     * Set birthday
     *
     * @param \DateTime $birthday
     * @return Person
     */
    public function setBirthday($birthday)
    {
        $this->birthday = $birthday;

        return $this;
    }

    /**
     * Get birthday
     *
     * @return \DateTime
     */
    public function getBirthday()
    {
        return $this->birthday;
    }

    /**
     * Add calendars
     *
     * @param Calendar $calendars
     * @return Person
     */
    public function addCalendar(Calendar $calendars)
    {
        $this->calendars[] = $calendars;

        return $this;
    }

    /**
     * Remove calendars
     *
     * @param Calendar $calendars
     */
    public function removeCalendar(Calendar $calendars)
    {
        $this->calendars->removeElement($calendars);
    }

    /**
     * Get calendars
     *
     * @return \Doctrine\Common\Collections\Collection
     */
    public function getCalendars()
    {
        return $this->calendars;
    }

    /**
     * @return string
     */
    public function getSalutation()
    {

    }
}
